cron.go

```diff
@@ -122,13 +106,13 @@ func WithErrorsHandler(f func(context.Context, Job, error)) CronOpt {
 	})
 }
 
-func WithEtcdMutexBuilder(b EtcdMutexBuilder) CronOpt {
+func WithEtcdClient(c *etcdclient.Client) CronOpt {
```

Comment:
Im now relying on this func: `WithEtcdMutexBuilder` in scheduler code:
```
	c, err := etcdcron.NewEtcdMutexBuilder(clientv3.Config{Endpoints: etcdEndpoints})
```
Once we switch to this library in scheduler code we will need to keep this change in mind.

Reply:
Yes, this is cleaner IMO. Compilation errors are the best way to remember to change :)